### PR TITLE
Ensure SQLAlchemy spans have kind set to CLIENT

### PR DESCRIPTION
**Pull request by owais**
# Description

SQLAlchemy spans were missing kind field and it was being set to
internal instead of client. This commit changes sqlalchemy spans to have
kind set to "client" instead.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Updated existing tests
- [x] Manual testing

# Does This PR Require a Core Repo Change?

- [ ] Yes. - Link to PR: 
- [x] No.

# Checklist:

See [contributing.md](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/master/CONTRIBUTING.md) for styleguide, changelog guidelines, and more.

- [x] Followed the style guidelines of this project
- [x] Changelogs have been updated
- [x] Unit tests have been added
- [ ] Documentation has been updated

### CODE REVIEW

File: opentelemetry-python-core [hunk 1]
```diff
@@ -0,0 +1 @@
+../opentelemetry-python
```
Comment (codeboten) [inline opentelemetry-python-core:1]: Is this file expected as part of this change?

**Review by codeboten — COMMENTED** [review]
Change looks good, just waiting on an answer to my question before approving.

File: opentelemetry-python-core (hunk 1 above)
Comment (owais) [reply opentelemetry-python-core:1]: No, this is not expected. This was supposed to be a draft PR. I probably created a regular PR by accident. Will fix in the morning.

Comment (owais) [reply opentelemetry-python-core]: Removed it.

**Pull request merged**